DatadogRUM/Tests/RUMMonitor/Scopes/RUMApplicationScopeTests.swift

```diff
@@ -208,61 +214,65 @@ class RUMApplicationScopeTests: XCTestCase {
     func testGivenStoppedSession_whenUserActionEvent_itStartsANewSession() throws {
         // Given
         let currentTime = Date()
-        let scope = RUMApplicationScope(
+        let scope = createRUMApplicationScope(
             dependencies: .mockWith(
                 sessionSampler: .mockKeepAll()
             )
         )
-        XCTAssertTrue(scope.process(
+        _ = scope.process(
```

Comment:
Why not continuing to assert the return value?

Reply:
The [contract from `RUMScope` interface](https://github.com/DataDog/dd-sdk-ios/blob/8aab1db2b7043435515335d45c5188825665396a/DatadogRUM/Sources/RUMMonitor/RUMScope.swift#L10-L15) is to use return type for signalling if a scope should be kept or released by parent scope. Because `RUMApplicationScope` is the root scope retained by `Monitor`, [we ignore its return type in `Monitor`](https://github.com/DataDog/dd-sdk-ios/blob/ncreated/RUM-1335/rum-session-start-reason/DatadogRUM/Sources/RUMMonitor/Monitor.swift#L138):

```swift
_ = self.scopes.process(command: transformedCommand, context: context, writer: writer)
```

Because we don't care about its return value, asserting this in tests was giving false sense of coverage. WDYT?